ui/app/templates/permission_scheduled_changes.html

```diff
@@ -42,11 +42,11 @@
     <h3 class="panel-title">
       <div style="float: right">
         <i ng-if=" scheduled_changes_rules_count && $first && (currentPage == 1)">Current</i>
-        <button class="btn btn-xs btn-primary" ng-click="signoff(sc)"
+        <button ng-show="!scheduled_changes_rules_count" class="btn btn-xs btn-primary" ng-click="signoff(sc)"
```

Comment:
With this applied, the Signoff and Revoke buttons are _always_ shown on pages like http://localhost:8080/rules/scheduled_changes, probably because you've added a new ng-show that has overriden the existing one for all of these buttons. You need to integrate this condition with the ones from the existing "ng-show" attributes, such as the one on line 46.